src/dps-notification-service/pom.xml

```diff
@@ -2,14 +2,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <parent>
-        <groupId>ca.bc.gov.open</groupId>
-        <artifactId>dps</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
-        <relativePath>
-            ../pom.xml
-        </relativePath>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.2.4.RELEASE</version>
+        <relativePath/>
```

Comment:
does this need to be included?

Reply:
yes, that force to reference the parents